internal/core/domain/market.go

```diff
@@ -104,6 +104,12 @@ func NewMarket(
 	if !isValidPrecision(quoteAssetPrecision) {
 		return nil, ErrMarketInvalidQuoteAssetPrecision
 	}
+	if !isValidStrategy(strategyType) {
+		return nil, ErrMarketUnknownStrategy
+	}
+	if strategyType == StrategyTypeUndefined {
+		strategyType = StrategyTypeBalanced
```

Comment:
here u set type to StrategyTypeBalanced if "undefined/unspecified" and in handler/utils.go if it is StrategyType_STRATEGY_TYPE_UNSPECIFIED u set type to StrategyTypePluggable?

Reply:
https://github.com/tdex-network/tdex-daemon/pull/708#discussion_r1183576891